utils/resource-monitor/README.md

```diff
@@ -1,22 +1,32 @@
 # Resource performance monitoring and graphing module
 
-This is a module for monitoring the resource usage of batch mode with HAPI as the source. The scripts in this module generate graphs and csv files of resource usage (CPU, memory and I/O) of the HAPI server, postgres database and pipeline over the duration of the batch job. The user is able to specify the number of processes/cores used in the pipeline to assess batch mode's performance on the local machine.
+This is a module for monitoring the resource usage of the batch pipeline with [HAPI](https://hapifhir.io/) 
+as the source. The scripts in this module generate graphs and csv files of resource usage 
+(CPU, memory and I/O) of the HAPI server, postgres database and pipeline over the duration of the 
+batch job. The user is able to specify the number of processes/cores used in the pipeline to assess 
+the batch pipeline's performance on the local machine.
 
 ## Prerequisites
 
--   Ensure you are able to run the `docker` command as a non-root user. If not, run <br />
+-   Ensure you are able to run the `docker` command 
+[as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/). If not, run  
 `sudo chmod 666 /var/run/docker.sock`
--   A [HAPI](https://hapifhir.io/) instance with a [postgres](https://www.postgresql.org/) instance as its database. For convenience, [`this`](/docker/hapi-compose.yml) .yml file in the in the [`docker`](/docker) directory contains images of a HAPI server using postrges as its database. Start the docker container with <br /> `docker-compose -f hapi-compose.yml up` 
+-   A [HAPI](https://hapifhir.io/) instance with a [postgres](https://www.postgresql.org/) instance 
+as its database. For convenience, [`this`](/docker/hapi-compose.yml) .yml file in the in the 
+[`docker`](/docker) directory contains images of a HAPI server using postrges as its database. Start 
+the docker container with  
+`docker-compose -f hapi-compose.yml up` 
```

Comment:
I think we should also make a brief reference to the uploader for preparing some data in the FHIR server. Also, we should either update the `hapi.compose.yml` to load a DB dump before bringing up HAPI or create docker images with data preloaded. This can be done in a separate PR if you prefer (but please add a TODO somewhere for it).

Reply:
I've added a brief reference to the uploader in the documentation. And with respect to creating docker images with data preloaded, I will work on issue #271 and create a separate PR.